app/styles/ooyala/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'ooyala-v4';
+@import 'article-featured-video';
```

Comment:
It looks like article component, not Ooyala one. Can we move it from here to the rest of article components?